src/qibo/backends/_clifford_operations.py

```diff
@@ -454,11 +499,32 @@ def cast(x, dtype=None, copy=False):
     return np.array(x, dtype=dtype, copy=copy)
 
 
-def _clifford_pre_execution_reshape(state):
+def _clifford_pre_execution_reshape(state, pack: bool = False):
+    """Reshape and packing applied to the symplectic matrix before execution to prepare the state in the form needed by each engine.
+
+    Args:
+        state (np.array): Input state.
+        pack (bool): Whether to pack the bits of the symplectic matrix.
+
+    Returns:
+        (np.array) The packed and reshaped state.
+    """
+    if pack:
+        state = _packbits(state, axis=0)
     return state
 
 
-def _clifford_post_execution_reshape(state, nqubits):
+def _clifford_post_execution_reshape(state, nqubits: int):
```

Comment:
This is asymmetric with respect to the one above, i.e. there is no `pack` flag (and it is always unpacking).

What's the reason?

Reply:
I double checked and indeed the pack arg can be dropped